src/js/components/stock-movement-wizard/modals/EditPickModal.jsx

```diff
@@ -11,17 +11,19 @@ import TextField from '../../form-elements/TextField';
 import ArrayField from '../../form-elements/ArrayField';
 import { renderFormField } from '../../../utils/form-utils';
 import SelectField from '../../form-elements/SelectField';
-import { REASON_CODE_MOCKS } from '../../../mockedData';
 import apiClient from '../../../utils/apiClient';
-import { showSpinner, hideSpinner } from '../../../actions';
+import { showSpinner, hideSpinner, fetchReasonCodes } from '../../../actions';
 
 const FIELDS = {
   reasonCode: {
     type: SelectField,
     label: 'Reason code',
     attributes: {
-      options: REASON_CODE_MOCKS,
+      required: true,
```

Comment:
Why you readded this? I removed this on purpose (rebase if you didnt do this before push)